product-service/serverless.ts

```diff
@@ -0,0 +1,96 @@
+import type { AWS } from '@serverless/typescript';
+
+import hello from '@functions/hello';
+import getProductsList from '@functions/getProductsList';
+import getProductsById from '@functions/getProductsById';
+import createProductHandler from '@functions/createProduct'
+import catalogBatchProcess from '@functions/catalogBatchProcess'
+
+const serverlessConfiguration: AWS = {
+  service: 'product-service',
+  frameworkVersion: '3',
+  plugins: ['serverless-esbuild'],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs14.x',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      PRODUCTS_TOPIC_ARN: {
+        Ref: 'ProductsTopic'
+      },
+      NODE_OPTIONS: '--enable-source-maps --stack-trace-limit=1000',
+      DB_NAME: 'cloud_x',
```

Comment:
Those have to be secrets :( You can use useDotEnv and thus hide them all in your .env file. With 'PG' you can create several .env files which you can later use for different purposes.

Reply:
Thanks @IvanRadchenko  for suggestion. I update code and moved these in .env file.
https://github.com/shekharverma17/shop-backend-services-aws/commit/e4667cdc9c0475871f14a192dc1d605fd2d0b072